ui/core-react/src/core-react/icons/ConditionalIconItem.tsx

```diff
@@ -27,6 +27,16 @@ export class ConditionalIconItem {
     this._value = value;
   }
 
+  /** A work-around for isInstanceOf */
+  public static isConditionalIconItem(item: any): boolean {
```

Comment:
Do you know why `ConditionalIconItem` required a workaround but `ConditionalStringValue` did not ?

I wonder if this could be related to the same issue I just had with `appui-test-providers` that wasnt build using esm and was breaking some use (I see that our example is in this package) Do we have an example of `ConditionalStringValue` in the providers ?